### PR TITLE
updated version and way to backport

### PR DESCRIPTION
**Pull request by davyzhang**

### CODE REVIEW

File: chromedriver/Dockerfile [hunk 1]
```diff
@@ -18,13 +19,13 @@ RUN mkdir -p /usr/share/man/man1 \
...
 RUN  mkdir -p /opt/selenium \
-  && wget --no-verbose https://selenium-release.storage.googleapis.com/3.0/selenium-server-standalone-3.0.1.jar -O /opt/selenium/selenium-server-standalone.jar
+  && wget --no-verbose https://selenium-release.storage.googleapis.com/3.3/selenium-server-standalone-3.3.1.jar -O /opt/selenium/selenium-server-standalone.jar
 
 
 #==================
 # Chrome webdriver
 #==================
-ARG CHROME_DRIVER_VERSION=2.27
+ARG CHROME_DRIVER_VERSION=2.9
```
Comment (yukinying) [inline chromedriver/Dockerfile]: Do you mean CHROME_DRIVER_VERSION=2.29?

Comment (yukinying) [inline chromedriver/Dockerfile]: Nevermind, I will merge and then change it to 2.29.

**Comment by yukinying** [comment]
I changed the version to 2.29 and the image is pushed to https://hub.docker.com/r/yukinying/chrome-headless-webdriver. Thank you!

**Comment by davyzhang** [comment]
But unfortunately, this approach won't work with selenium standalone mode, I still can not figure it out how to use it with grid and node configure. So I drop it and turn to chrome debug protocol here at https://github.com/wirepair/gcd, I also see your issues there 😄 

Cheers

**Comment by yukinying** [comment]
Could you describe what issue you see with the standalone mode there? It's been a while since I reported an issue on gcd (a year back). I have tested the chrome-headless-webdriver image with Protractor and it is working with some basic test. I may need to extend some of the tests before I could publish how it is done. If you could give me some feedback on your use case or issues you seen so far, it would be very helpful. Thank you!

**Comment by davyzhang** [comment]
I run the docker with
`docker run  --rm -it --shm-size=1024m --cap-add=SYS_ADMIN -p 4444:4444 chromedriver`

test with a simple python script

```python
from selenium import webdriver
from selenium.webdriver.common.desired_capabilities import DesiredCapabilities

browser = webdriver.Remote(
   command_executor='http://127.0.0.1:4444/wd/hub',
   desired_capabilities=DesiredCapabilities.CHROME)
   
browser.get("http://www.google.com/ncr")

browser.save_screenshot("test.png")
browser.quit()
```

and here is the error and log

```
03:05:53.808 INFO - Selenium build info: version: '3.3.1', revision: '5234b32'
03:05:53.810 INFO - Launching a standalone Selenium Server
2017-04-18 03:05:53.841:INFO::main: Logging initialized @322ms to org.seleniumhq.jetty9.util.log.StdErrLog
03:05:53.925 INFO - Driver provider org.openqa.selenium.ie.InternetExplorerDriver registration is skipped:
 registration capabilities Capabilities [{ensureCleanSession=true, browserName=internet explorer, version=, platform=WINDOWS}] does not match the current platform LINUX
03:05:53.926 INFO - Driver provider org.openqa.selenium.edge.EdgeDriver registration is skipped:
 registration capabilities Capabilities [{browserName=MicrosoftEdge, version=, platform=WINDOWS}] does not match the current platform LINUX
03:05:53.926 INFO - Driver class not found: com.opera.core.systems.OperaDriver
03:05:53.927 INFO - Driver provider com.opera.core.systems.OperaDriver registration is skipped:
Unable to create new instances on this machine.
03:05:53.928 INFO - Driver class not found: com.opera.core.systems.OperaDriver
03:05:53.928 INFO - Driver provider com.opera.core.systems.OperaDriver is not registered
03:05:53.930 INFO - Driver provider org.openqa.selenium.safari.SafariDriver registration is skipped:
 registration capabilities Capabilities [{browserName=safari, version=, platform=MAC}] does not match the current platform LINUX
2017-04-18 03:05:53.985:INFO:osjs.Server:main: jetty-9.2.20.v20161216
2017-04-18 03:05:54.029:INFO:osjsh.ContextHandler:main: Started o.s.j.s.ServletContextHandler@604ed9f0{/,null,AVAILABLE}
2017-04-18 03:05:54.057:INFO:osjs.AbstractConnector:main: Started ServerConnector@6fd02e5{HTTP/1.1,[http/1.1]}{0.0.0.0:4444}
2017-04-18 03:05:54.058:INFO:osjs.Server:main: Started @540ms
03:05:54.059 INFO - Selenium Server is up and running
03:05:56.062 INFO - SessionCleaner initialized with insideBrowserTimeout 0 and clientGoneTimeout 1800000 polling every 180000
03:05:56.650 INFO - Executing: org.openqa.selenium.remote.server.handler.GetAllSessions@1d632b4e)
03:05:56.657 INFO - Done: org.openqa.selenium.remote.server.handler.GetAllSessions@1d632b4e
03:06:14.767 INFO - Executing: [new session: Capabilities [{browserName=chrome, javascriptEnabled=true, version=, platform=ANY}]])
03:06:14.801 INFO - Creating a new session for Capabilities [{browserName=chrome, javascriptEnabled=true, version=, platform=ANY}]
Starting ChromeDriver 2.29.461571 (8a88bbe0775e2a23afda0ceaf2ef7ee74e822cc5) on port 13689
Only local connections are allowed.
03:07:15.346 INFO - Detected dialect: OSS
03:07:15.377 INFO - Done: [new session: Capabilities [{browserName=chrome, javascriptEnabled=true, version=, platform=ANY}]]
03:07:15.385 INFO - Executing: [get: http://www.google.com/ncr])
03:07:15.430 WARN - Exception thrown
org.openqa.selenium.NoSuchSessionException: no such session
  (Driver info: chromedriver=2.29.461571 (8a88bbe0775e2a23afda0ceaf2ef7ee74e822cc5),platform=Linux 4.9.13-moby x86_64) (WARNING: The server did not provide any stacktrace information)
Command duration or timeout: 9 milliseconds
Build info: version: '3.3.1', revision: '5234b32', time: '2017-03-10 09:04:52 -0800'
System info: host: '561b186dbd72', ip: '172.17.0.2', os.name: 'Linux', os.arch: 'amd64', os.version: '4.9.13-moby', java.version: '1.8.0_121'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Capabilities [{message=unknown error: Chrome failed to start: exited abnormally
  (Driver info: chromedriver=2.29.461571 (8a88bbe0775e2a23afda0ceaf2ef7ee74e822cc5),platform=Linux 4.9.13-moby x86_64), platform=ANY}]
Session ID: ecf568800629c79568db522e7e30de4b
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
	at org.openqa.selenium.remote.ErrorHandler.createThrowable(ErrorHandler.java:216)
	at org.openqa.selenium.remote.ErrorHandler.throwIfResponseFailed(ErrorHandler.java:168)
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:638)
	at org.openqa.selenium.remote.RemoteWebDriver.get(RemoteWebDriver.java:325)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.openqa.selenium.support.events.EventFiringWebDriver$2.invoke(EventFiringWebDriver.java:104)
	at com.sun.proxy.$Proxy4.get(Unknown Source)
	at org.openqa.selenium.support.events.EventFiringWebDriver.get(EventFiringWebDriver.java:163)
	at org.openqa.selenium.remote.server.handler.ChangeUrl.call(ChangeUrl.java:40)
	at org.openqa.selenium.remote.server.handler.ChangeUrl.call(ChangeUrl.java:25)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at org.openqa.selenium.remote.server.DefaultSession$1.run(DefaultSession.java:176)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
03:07:15.433 WARN - Exception: no such session
  (Driver info: chromedriver=2.29.461571 (8a88bbe0775e2a23afda0ceaf2ef7ee74e822cc5),platform=Linux 4.9.13-moby x86_64) (WARNING: The server did not provide any stacktrace information)
Command duration or timeout: 9 milliseconds
Build info: version: '3.3.1', revision: '5234b32', time: '2017-03-10 09:04:52 -0800'
System info: host: '561b186dbd72', ip: '172.17.0.2', os.name: 'Linux', os.arch: 'amd64', os.version: '4.9.13-moby', java.version: '1.8.0_121'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Capabilities [{message=unknown error: Chrome failed to start: exited abnormally
  (Driver info: chromedriver=2.29.461571 (8a88bbe0775e2a23afda0ceaf2ef7ee74e822cc5),platform=Linux 4.9.13-moby x86_64), platform=ANY}]
Session ID: ecf568800629c79568db522e7e30de4b
```

**Comment by yukinying** [comment]
I have made a couple changes: 

1. Modified the [Dockerfile](https://github.com/yukinying/chrome-headless-browser-docker/blob/master/chromedriver/Dockerfile) with https://github.com/yukinying/chrome-headless-browser-docker/commit/a3cdb45e53f6c3d684c9781122a8444600731ec6
2. Renamed the docker image to [yukinying/chrome-headless-browser-selenium](https://hub.docker.com/r/yukinying/chrome-headless-browser-selenium/)
3. Updated the Readme.MD for the start instruction, i.e. 
```
docker run -it --rm --name chrome --shm-size=1024m --cap-add=SYS_ADMIN \
  -p=127.0.0.1:4444:4444 \
  yukinying/chrome-headless-browser-selenium
```

I have tested your script above, it works without any issues.

**Pull request merged**